packages/tree/Tree.scss

```diff
@@ -0,0 +1,5 @@
+@import "~blaze/scss/components.trees";
+
+ul {
+  padding-left: 1em !important;
```

Comment:
No, because when someone (like us in demo) use jsx instead of html web element, this selector does not match.